src/components/Hero/heroChangelogData.ts

```diff
@@ -9,6 +9,16 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Styles"],
+    notes: [
+      "Updates the `primary` variant of the `Hero` component to have appropriate padding for desktop and mobile, and `max-width` of 860px",
+      "Updates the `campaign` variant of the `Hero component to have consistent padding.",
```

Comment:
You can remove reference to the "Hero component" in these entries. For example...
```
Updates the `primary` variant to have appropriate padding 
for desktop and mobile, and `max-width` of 860px.
```